### ISSUE TITLE
Issue with filtering by time

### ISSUE THREAD

**Issue by ordirules** [NSLS-II/databroker#147]
Hi,

  I'm trying out filters, available [v 0.6 onwards](https://nsls-ii.github.io/databroker/searching.html).
  I'm running databroker 0.7.0+11.g71804f3 (latest fetched master branch as  of now).

The time filters don't seem to work as expected. For instance, in this code:

```python
import datetime
def timestring(unixtime):
    timestr = datetime.datetime.fromtimestamp(
        int(unixtime)).strftime('%Y-%m-%d %H:%M:%S')
    return timestr

#### Clear filters just in case
chxdb.clear_filters()
#### this search works
headers = chxdb(comment="H3 R360nm #1")
header1 = headers[0]
print("header 1 time : {}".format(header1['start']['time']))
print("header 1 time : {}".format(timestring(header1['start']['time'])))
#### add filters
chxdb.add_filter(start_time="2017-02-01")
chxdb.add_filter(stop_time = "2017-03-00")
#### This search returns nothing, even though time is within filter (see output)
headers = chxdb(comment="H3 R360nm #1")

header2 = headers[0]
print("header 2 time : {}".format(header2['start']['time']))
print("header 2 time : {}".format(timestring(header['start']['time'])))
```

The output is

    header 1 time : 1486929765.6251316
    header 1 time : 2017-02-12 15:02:45
    ---------------------------------------------------------------------------
    IndexError                                Traceback (most recent call last)
    /.../somefile.py in <module>()
         21 chxdb.add_filter(stop_time = "2017-03-00")
         22 headers = chxdb(comment="H3 R360nm #1")
    ---> 23 header2 = headers[0]
         24 print("header 2 time : {}".format(header2['start']['time']))
         25 print("header 2 time : {}".format(timestring(header['start']['time'])))

Basically, after adding running  `add_filter`, data is no longer found found. Other filters seem to work (`chxdb.add_filter(comment="foo")`, works). Seems to have something to do with time. thanks!

**Comment by danielballan** [comment]
Yeah, this looks related to #119. Thanks for the additional info.

**Comment by ordirules** [comment]
ok, I'll leave open just in case they're not 100% related. (or let me know if i should close this and paste the text in the #119)

**Comment by danielballan** [comment]
Cool. I agree this should stay open.

**Comment by danielballan** [comment]
Closed by #230. Thanks for the report, @ordirules.
